### ISSUE TITLE
Avatar Icon not visible

### ISSUE THREAD

**Issue by duke1712** [pchmn/MaterialChipsInput#2]
Hi,
I added this code in my adapter to add chips programatically, but I am not getting the letter tile when using this. There is a black space for the avatar icon.When I am doing this using xml then it is working perfectly.

```
ChipView chip=new ChipView(context);
            chip.setLabel(sms.getRecieverList().get(i).getLabel());
            chip.setLabelColor(context.getColor(R.color.colorPrimaryDark));
            chip.setAvatarIcon(sms.getRecieverList().get(i).getAvatarUri());
            chip.setHasAvatarIcon(true);
```

**Comment by pchmn** [comment]
Yes, the bug occurred because I guess that `sms.getRecieverList().get(i).getAvatarUri()` was `null`.

Nevermind, I fixed this bug and pushed the correction in v1.0.2 of the library. 

PS : if you use `setAvatarIcon(Uri uri)`, you don't have to specify `setHasAvatarIcon(true)`, it will be set automatically

Sorry for the late response

**Comment by duke1712** [comment]
Thank for the reply,
Actually when I am creating chip view programmatically, the avatar is not
coming.I just needed the letter icon beside each contact. Ya the Avatar
icon uri is null most if the time.

I will test it out and let you know.

Thanks again
